### ISSUE TITLE
Murasaki app not available

### ISSUE THREAD

**Issue by ndxyz00** [GenjiApp/EPUB-Plugins#9]
Murasaki app not available in the App store, at least in EU.
Also would it be possible to add other popular formats like mobi, djvu, etc ...

Thanks

**Comment by GenjiApp** [comment]
I decided not to publish Murasaki in the EU region a few days ago.
Under the EU's Digital Services Act (DSA), if I sell paid apps in the EU, my real address and phone number must be displayed on the App Store.
As an indie developer, I don’t want that.

**Comment by ndxyz00** [comment]
hi, i know regulations sucks,
would it be possible to sell through gumroad maybe?
thanx

**Comment by naga-re** [comment]
Really too bad for us, Europeans, but, in the long run, as for any commercial relationship on these digital platforms, this will also be the case elsewhere, in other countries, in the United States as well as in Japan. A customer must have a way to contact the seller of a good and the intermediary (here Apple) must ensure that the service/application provider is a real organization/person.
It's a classic. For some "scam" developers, all developers must accept new rules, to better protect the consumer.

But your statement is a little inaccurate. **DSA does not require you to give your personal address and your personal phone number.**

It asks for an email address (quite trivial to have a professional contact address), a contact phone number (no obligation to give yours) and a professional address/PO box.

In no case does the DSA require the "merchant" to provide personal information such as their home address or private telephone number.

Furthermore, from a legal point of view, this would contradict the European GDPR regulation on personal data.
Apple must collect elements that allow the identification of the "merchant", whether an organization or an independent developer, and publish them on the AppStore. But it does not have the right to publicly distribute and publish personal data: the address of a private home, a private telephone number are personal data.

And, indeed, as said in another comment, Gumroad could be one of the solutions.

**Comment by GenjiApp** [comment]
You are right; the DSA does not require a personal phone number or address.
However, obtaining and maintaining a new phone number and address comes with costs.
To be honest, the revenue from the App Store does not cover these costs.

About Gumroad:
I think that Gumroad, like the App Store, is an intermediary, isn't it?
I am not sure whether Gumroad falls within the scope of the DSA.
I contacted Gumroad; however, I have not received an answer yet.
